### PR TITLE
Added tests for tables command and testing docs

### PR DESCRIPTION
**Pull request by fredrikekelund**
Fixes https://github.com/Automattic/dotcom-forge/issues/9976

Following up from https://github.com/Automattic/wp-cli-sqlite-command/pull/5, this PR adds two Behat tests for the `wp sqlite tables` command.

Because I wasn't previously familiar with Behat, I also took the opportunity to add some rudimentary docs.

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+This repo uses the [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development) testing framework [Behat](https://docs.behat.org/en/latest/). It means we use plain text files to describe "scenarios" containing keywords like "given", "when", and "then". This methodology was [chosen](https://github.com/Automattic/wp-cli-sqlite-command/issues/3) because it's what WP CLI itself uses to [test commands](https://github.com/wp-cli/wp-cli-tests).
+
+The test cases are located in `features/*.feature`.
+
+## Running tests locally
+
+The test cases depend on real WordPress installations, meaning the usual PHP and MySQL dependencies are required. However, we can opt to use SQLite instead of MySQL to make running the tests locally easier. Here's how to run the full test suite with a SQLite database:
+
+```
+WP_CLI_TEST_DBTYPE=sqlite vendor/bin/behat
```
Comment (wojtekn) [inline TESTING.md:12]: Nice! I missed that before, and I was running those tests by providing a path to MySQL.

**Review by wojtekn — APPROVED** [review]
Thanks for documenting tests!

**Comment by fredrikekelund** [comment]
It doesn't look like we run these tests with CI 🤔 I'll take a quick look at that, too, but I might submit a fix separately

**Comment by fredrikekelund** [comment]
It looks like the [Testing workflow](https://github.com/Automattic/wp-cli-sqlite-command/actions/workflows/testing.yml) had been automatically disabled because [Github does that](https://github.com/orgs/community/discussions/13751) when a repo has been inactive for N days, and the action uses a cron schedule (which this action did).

I assume we'll be using pull requests to push new code to this repo at this point, meaning I don't see a clear point in running that workflow on a cron schedule. I removed the cron schedule to prevent the same thing from happening again.

**Pull request merged**